src/app/components/Header/Header.tsx

```diff
@@ -63,6 +92,12 @@ export default function Header() {
           />
         </div>
       </USWDSHeader>
-    </>
+      {customContent && (
```

Comment:
As I mentioned above, this check is correct 👍  changing the `SpecialContent` type should cause an error here if the  `customContent &&` check is removed.